### PR TITLE
Define monospace fonts more explicitly

### PR DESCRIPTION
**Pull request by Artanicus**
Define monospace fonts more explicitly to avoid rendering issues on various operating systems. Long story short, not all monospace fonts are created equal and their handling of wide unicode characters differs. I tried a lot of fonts and DejaVu Sans Mono handles well for Linux (and derivatives such as ChromeOS) and Menlo handles well for OS X. I don't have an opportunity to find a good one for Windows so I'll leave that as an exercise for someone who uses Windows.

The fallback of monospace is still there so this should only improve the situation for those having issues. See: #22

### CODE REVIEW

**Comment by chubin** [comment]
That's a good fix, thank you, but it does not fix the problem completely. I think the only solution that will cover 100% cases is to use embedded fonts. By the way, have you tested your fix with some other systems besides OS X? Android may be?

**Comment by Artanicus** [comment]
I tried going to embedded fonts, or rather to avoid a large download to link a font from fonts.google.com. Unfortunately it seems a large portion on monospace fonts actually fail at wide characters being monospace. I couldn't find a single font that would have worked. DejaVu Sans Mono is exclusive to Linux and derivatives but unfortunately doesn't seem to be available on Android. Menlo works beautifully with OS X but is very exclusive to OS X.

I agree that finding and web-linking one font that works for all platforms would be the best option, I just couldn't find a font that would fit the bill. In the end I gave up and found OS specific fonts that are actually monospace even with wide characters. I've seen Consolas recommended as being a proper monospace font but I can't test it myself.

TL;DR; yes, one font should rule them all, but first we need to find that font. The alternative is finding a good standard-shipped font from every major platform, missing are still:
- Android
- Windows
- iOS (may have Menlo? needs testing anyway.)

**Comment by chubin** [comment]
I've just tested your suggestion with iOS. It works really perfect, kudos :+1: 
But with Android, as I can see, the problem is still there.

**Comment by chubin** [comment]
Currently on wttr.in we use 

> font-family: DejaVu Sans Mono, Menlo, Lucida Sans Typewriter, Lucida Console, monaco, Bitstream Vera Sans Mono, monospace;

**Comment by Artanicus** [comment]
Sounds like we should open an issue to track what fonts are actually needed to support which platforms. A shotgun approach may just confuse things. There's still at least Android to figure out.

**Comment by chubin** [comment]
May be we should use #68 for that?

**Comment by Artanicus** [comment]
SGTM.
